docs/api/cli-options.md

```diff
@@ -76,6 +76,143 @@ Options include:
+    chromatic: ^10.2.0 => 10.2.0
+    msw-storybook-addon: ^1.10.0 => 1.10.0
+    storybook: ^7.6.6 => 7.6.6
+```
+
+### `remove`
+
+**Added in:** Storybook 8.0
+
+Deletes a Storybook addon from your project. Read more in the [addon installation guide](../addons/install-addons.md#removing-addons).
+
+```shell
+storybook remove [addon] [options]
+```
+
+Options include:
+
+| Option              | Description                                                                                                                                                                     |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `--package-manager` | Sets the package manager to use when removing the addon.<br/>Available package managers include `npm`, `yarn`, and `pnpm`<br/>`storybook remove  [addon]--package-manager pnpm` |
+
+### `sandbox`
+
+Generates a local sandbox project for testing Storybook features based on the list of supported [frameworks](../configure/frameworks.md). Useful for reproducing bugs when opening an issue or a discussion.
+
+```shell
+storybook sandbox [framework-filter] [options]
+```
+
+<Callout variant="info">
+
+The `framework-filter` argument is optional and can filter the list of available frameworks. For example, `storybook sandbox react` will only show React-based sandboxes.
+
+</Callout>
+
+Options include:
+
+| Option                         | Description                                                                                                                                                                                                                       |
+| ------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `-b`, `--branch [branch name]` | Select the branch to use for the sandbox project featuring the available features present in the selected branch (`next` for the pre-release version, `main` for the latest stable release)<br/>`storybook sandbox --branch main` |
+| `-o`, `--output [dir-name]`    | Configures the location of the sandbox project<br/>`storybook sandbox --output /my-sandbox-project`                                                                                                                               |
+| `--no-init`                    | Generates a sandbox project without without initializing Storybook<br/>`storybook sandbox --no-init`                                                                                                                              |
+
+<Callout variant="info">
+
+If you're looking for a hosted version of the available sandboxes, see [storybook.new](https://new-storybook.netlify.app/).
```

Comment:
Shouldn't this link to `https://storybook.new`?

Reply:
Thanks for calling that out, I'll adjust accordingly.